src/test/java/pageObject/test/PageObjectLogin.java

```diff
@@ -0,0 +1,22 @@
+package pageObject.test;
+
+import org.junit.jupiter.api.Test;
+import pageObject.BaseEntities.BaseTest;
+import pageObject.pages.InMailPageFactory;
+import pageObject.pages.LoginPageFactory;
+
+public class PageObjectLogin extends BaseTest {
```

Comment:
NAMING. what do you test here, Login, correct, so, the class name should be smthing like LoginTest